ci/create_database.py

```diff
@@ -107,6 +110,26 @@ def generate_token(size=12):
     })
 
 
+did_shutdown = False
+
+
+async def shutdown():
```

Comment:
Why is did_shutdown a global variable and who is it used by?

Reply:
Shutdown is called inside the migrate function which is called once for each migration (in order).  Migration might already be applied, in which case the function does nothing.  We only shut down if we actually apply a migration, in which case shutdown is called.  But we only need to shutdown once, so did_shutdown records whether we've already shutdown.